apps/chat/src/components/Promptbar/components/PromptbarSettings.tsx

```diff
@@ -36,8 +38,29 @@ export function PromptbarSettings() {
     PromptsSelectors.selectDoesAnyMyItemExist,
   );
 
+  const isSelectMode = useAppSelector(PromptsSelectors.selectIsSelectMode);
+
+  const deleteTerm = isSelectMode ? 'selected' : 'all';
```

Comment:
Maybe lets move this to enum? Like `DeleteTermEnum`

Reply:
I don't think it's good idea, because it's just different UI text, not more